sxbp/render_backends/backend_png.c

```diff
@@ -166,8 +166,8 @@ sxbp_status_t sxbp_render_backend_png(
     metadata[2].key = "Copyright";
     metadata[2].text = "Copyright Joshua Saxby";
     metadata[3].key = "Software";
-    // SAXBOSPIRAL_VERSION_STRING is a macro that expands to a double-quoted string
-    metadata[3].text = "libsxbp v" SAXBOSPIRAL_VERSION_STRING;
+    // LIBSXBP_VERSION_STRING is a macro that expands to a double-quoted string
```

Comment:
I was wondering a bit why this was not causing errors before. It seems to be because undefined identifiers take on the value of 0 by default in C.